Library/Formula/hue.rb

```diff
@@ -0,0 +1,57 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
```

Comment:
Good point but gethue.com does not have https yet, it will break the page on modern browsers